platform-grpc-service-framework/src/main/java/org/hypertrace/core/serviceframework/grpc/GrpcPlatformServiceContainer.java

```diff
@@ -199,6 +199,23 @@ public boolean healthCheck() {
     }
   }
 
+  private Duration[] generateLatencyHistogramBuckets() {
+    ArrayList<Duration> histogram_buckets = new ArrayList<>();
+    int factor = 3;
+    long curr_duration_ms =
+        getAppConfig().hasPath(MAX_LATENCY_HISTOGRAM_BUCKET_MS)
+            ? getAppConfig().getLong(MAX_LATENCY_HISTOGRAM_BUCKET_MS)
+            : DEFAULT_MAX_LATENCY_HISTOGRAM_BUCKET_MS;
+    while (curr_duration_ms >= 10) {
+      histogram_buckets.add(Duration.ofMillis(curr_duration_ms));
```

Comment:
Ah so we're converting it back to duration anyway - even more of a reason to just work strictly in durations.